test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -673,6 +673,116 @@ context('action sidebar', function() {
       });
   });
 
+  specify('action comments', function() {
+    cy
+      .server()
+      .routePatient()
+      .routePatientActions()
+      .routePatientFlows()
+      .routeAction(fx => {
+        fx.data.id = '12345';
+        fx.data.relationships['program-action'] = { data: { id: '1' } };
+        fx.data.relationships.form = { data: { id: '11111' } };
+        return fx;
+      })
+      .routeActionActivity(fx => {
+        fx.data = [];
+        fx.data[0] = this.fxEvents[0];
+        fx.data[1] = this.fxEvents[1];
+        fx.data[0].relationships.editor.data = null;
+        fx.data[0].attributes.date = now.format();
+
+        fx.data.push({
+          id: '12345',
+          type: 'events',
+          attributes: {
+            date: now.format(),
+            type: 'ActionCopiedFromProgramAction',
+          },
+          relationships: {
+            'program': { data: { id: '1' } },
+            'program-action': { data: { id: '1' } },
+            'editor': { data: { id: '11111' } },
+          },
+        });
+
+        return fx;
+      })
+      .routeActionComments(fx => {
+        fx.data = _.sample(fx.data, 3);
+
+        fx.data[0].relationships.clinician.data = { id: '11111' };
+        fx.data[0].attributes.edited_at = null;
+        fx.data[0].attributes.updated_at = moment(now).subtract(2, 'days').format();
+        fx.data[0].attributes.message = 'Least Recent Message from Clinician McTester';
+
+        fx.data[1].relationships.clinician.data = { id: '11111' };
+        fx.data[1].attributes.edited_at = now.format();
+        fx.data[1].attributes.updated_at = now.format();
+        fx.data[1].attributes.message = 'Most Recent Message from Clinician McTester';
+
+        fx.data[2].relationships.clinician.data = { id: '22222' };
+        fx.data[2].attributes.updated_at = moment(now).subtract(4, 'days').format();
+        fx.data[2].attributes.message = 'Message from Someone Else';
+        fx.data[2].attributes.edited_at = null;
+
+
+        return fx;
+      })
+      .routeProgramByAction(fx => {
+        fx.data.id = '1';
+        fx.data.attributes.name = 'Test Program';
+
+        return fx;
+      })
+      .routePrograms()
+      .routeAllProgramActions()
+      .routeAllProgramFlows()
+      .visit('/patient/1/action/12345')
+      .wait('@routePatient')
+      .wait('@routePatientActions')
+      .wait('@routeAction')
+      .wait('@routeActionActivity')
+      .wait('@routeActionComments')
+      .wait('@routeProgramByAction');
+
+    cy
+      .get('[data-activity-region]')
+      .find('.u-margin--b-8')
```

Comment:
How about `td` instead of a utility class.  or add a `qa-activity-item` class.  I don't think we should test off of utility classes if it is avoidable.